app_server/app/Router.js

```diff
@@ -83,7 +85,6 @@ server.register({
   }
 });
 
-/* Register controllers */
```

Comment:
This is intentional, right?

Also, are we going to swap `Authenticator` with `server.app.authenticator` globally? May be good the mass replace in this PR to reduce debt later on.

Aside from those, the rest look fine, go ahead and merge.